idea_town/experiments/migrations/0001_initial.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+import idea_town.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Experiment',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', serialize=False, primary_key=True)),
+                ('title', models.CharField(max_length=128)),
+                ('slug', models.SlugField(unique=True, max_length=128)),
+                ('thumbnail', models.ImageField(upload_to=idea_town.utils.HashedUploadTo('thumbnail'))),
+                ('description', models.TextField()),
+                ('created', models.DateTimeField(auto_now_add=True)),
+                ('modified', models.DateTimeField(auto_now=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='ExperimentDetail',
+            fields=[
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', serialize=False, primary_key=True)),
+                ('order', models.IntegerField(default=0)),
+                ('headline', models.CharField(max_length=256)),
+                ('image', models.ImageField(upload_to=idea_town.utils.HashedUploadTo('image'))),
```

Comment:
Yup, that should be a TODO. I started looking into a couple Django thumbnail apps to maybe take care of this easily, but hadn't decided on one yet. I think one of them does cached on-demand resizing from URL parameters, so that might be handy. Or, we use another one with sizes decided up front and push pre-generated thumbnails to S3 or something